rosdep/base.yaml

```diff
@@ -3294,6 +3297,7 @@ nodejs-legacy:
 npm:
   debian: [npm]
   fedora: [npm]
+  gentoo: [net-libs/nodejs]
```

Comment:
@Alessandro-Barbieri I, unfortunately, do not know the `USE` flags required for most ROS packages, and don't see it as a worth-while endeavor to go through every pack by hand and determine it... I also can't think of a good way to script it. So I think my standpoint here will be enough to be Ubuntu-esque, and, since Ubuntu has switched to Gnome 3, I think that can be relaxed to "enough to support Gnome 3."I